geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/CreateRegionCommand.java

```diff
@@ -243,9 +243,9 @@ public ResultModel createRegion(
 
       if (!colocatedRegionBean.getRegionType().equals("PARTITION") &&
           !colocatedRegionBean.getRegionType().equals("PERSISTENT_PARTITION")) {
-        return ResultModel.createError(CliStrings.format(
+        return ResultModel.createError(String.format(
             CliStrings.CREATE_REGION__MSG__COLOCATEDWITH_REGION_0_IS_NOT_PARTITIONEDREGION,
-            new String[] {prColocatedWith}));
+            (Object) prColocatedWith));
```

Comment:
looks like you have a failing test with this. It's much simpler to use String.format in this case.